version.json

```
@@ -0,0 +1,25 @@
+{
+  "$schema": "https://raw.githubusercontent.com/dotnet/Nerdbank.GitVersioning/main/src/NerdBank.GitVersioning/version.schema.json",
+  "version": "2024.0.0",
+  "publicReleaseRefSpec": [
+      "^refs/heads/master$", // we release out of master
+      "^refs/tags/\\d{4}\\.\\d+\\.\\d+(-beta\\d+)?$" // we also release tags starting with year.minor.build[-betaN]
+  ],
+  "cloudBuild": {
+    "buildNumber": {
+      "enabled": true
+    }
+  },
+  "assemblyVersion": {
+    "precision": "minor"
+  },
+  "nugetPackageVersion": {
+   "precision": "minor"
+  },
+  "release": {
+    "tagName" : "{version}",
+    "branchName" : "master",
```

I think this should be e.g.

```suggestion
    "branchName" : "release/{version}",
```

---

I think the curernt release is based on `master` branch with tag?